torch/csrc/jit/runtime/operator.h

```diff
@@ -66,26 +66,25 @@ struct TORCH_API Operator {
   Operator(c10::OperatorHandle opHandle, Operation operation)
       : schema_(std::make_shared<FunctionSchema>(opHandle.schema())),
         op_(std::make_shared<Operation>(std::move(operation))),
-        c10Handle_(opHandle),
```

Comment:
note: any changes in operator.h are likely to collide with https://github.com/pytorch/pytorch/pull/33905

Reply:
I took a look, the merge conflict looks easy to resolve.